eng/tools/spec-gen-sdk-runner/src/utils.ts

```diff
@@ -178,7 +178,7 @@ export function findParentWithFile(
     try {
       const absolutePath = path.resolve(specRepoFolder, currentPath);
       const files = fs.readdirSync(absolutePath);
-      if (files.some((file) => searchFile.test(file.toLowerCase()))) {
+      if (files.some((file) => searchFile.test(file))) {
```

Comment:
Fine for now, but eventually I think this should move to (or use an existing) shared JS helper.